google-cloud-logging/src/test/java/com/google/cloud/logging/BaseSystemTest.java

```diff
@@ -335,26 +336,27 @@ public void testListMetricsAsync() throws ExecutionException, InterruptedExcepti
 
   @Test
   public void testWriteAndListLogEntries() throws InterruptedException {
-    String logName = formatForTest("test-write-log-entries-log");
-    String filter =
-        "logName = projects/" + logging().getOptions().getProjectId() + "/logs/" + logName;
+    String logId = formatForTest("test-write-log-entries-log");
+    LoggingOptions loggingOptions = logging().getOptions();
+    LogName logName = LogName.create(loggingOptions.getProjectId(), logId);
     StringPayload firstPayload = StringPayload.of("stringPayload");
     LogEntry firstEntry = LogEntry.newBuilder(firstPayload)
         .addLabel("key1", "value1")
-        .setLogName(logName)
+        .setLogName(logId)
```

Comment:
It seems like `logName` should be passed here - I think the tests have been passing the wrong thing. 

https://github.com/googleapis/googleapis/blob/master/google/logging/v2/log_entry.proto#L39